vmod/vmod_vtc.c

```diff
@@ -372,3 +372,43 @@ vmod_proxy_header(VRT_CTX, VCL_ENUM venum, VCL_IP client, VCL_IP server,
 	return (VRT_blob(ctx, "proxy_header", h, l,
 	    BLOB_VMOD_PROXY_HEADER_TYPE));
 }
+
+/*--------------------------------------------------------------------*/
+
+VCL_DURATION v_matchproto_(td_vtc_benchmark)
+vmod_benchmark(VRT_CTX, VCL_SUB sub, VCL_INT total, VCL_INT leg)
+{
```

Comment:
Please leave this out of this PR.

We probably want something like this, but it needs to report proper statistics (min/max/avg/stddev and probably median)

Reply:
removed in the next force-push